eos/utils/expression-parser_TEST.cc

```diff
@@ -177,46 +214,50 @@ class ExpressionParserTest :
 
             // testing cloning and usage of parameters
             {
-                ExpressionTest test("6 * <<mass::mu>> - <<mass::tau>>");
+                ExpressionTest test("{q2} - 4 * <<mass::mu>> ^ 2");
```

Comment:
I don't like simplifying the test case. Previously, you tested non-trivially for two distinct parameters (Ids). Can you use ``<<mass::mu>> * <<mass::tau>>`` instead?